openeogeotrellis/job_tracker.py

```diff
@@ -170,7 +177,46 @@ def update_statuses(self) -> None:
                                 if final_state != "UNDEFINED":
                                     result_metadata = self._batch_jobs.get_results_metadata(job_id, user_id)
                                     # TODO: skip patching the job znode and read from this file directly?
-                                    registry.patch(job_id, user_id, **result_metadata)
+
+                                    sentinelhub_processing_units = (result_metadata.get("usage", {})
```

Comment:
would it be possible to refactor this sentinelhub logic to a toplevel function, so I can easily reuse it from job_tracker_v2?

Reply:
I'll create another ticket to address this along with some other small things.